tests/roundware/api1/test_commands.py

```diff
@@ -19,7 +19,9 @@
                                      mock_distance_in_meters_far)
 from roundware.lib.exception import RoundException
 from roundware.api1.commands import (check_for_single_audiotrack, get_asset_info,
-                                     get_available_assets)
+                                     get_available_assets,get_parameter_from_request,
+op_log_event,play_asset_in_stream,get_config,get_tags_for_project,get_events,current_version)
```

Comment:
Fix spacing. The o in `op_log_event` should be aligned under the g in `get_available_assets`